### PR TITLE
[REVIEW] Upgrade min `dask` & `distributed` versions

### PR DESCRIPTION
**Pull request by galipremsagar**
This PR upgrades minimum versions of `dask` & `distributed`. Upstream changes have been made: https://github.com/rapidsai/cudf/pull/10182/

### CODE REVIEW

**Comment by pentschev** [comment]
We will still need a new release of Distributed and https://github.com/rapidsai/dask-cuda/pull/870 to be merged, those were breaking changes.

**Comment by galipremsagar** [comment]
> We will still need a new release of Distributed and #870 to be merged, those were breaking changes.

This means we will require to upgrade min version to `2022.03.1` as opposed to `2022.02.1`?

**Comment by galipremsagar** [comment]
rerun tests

**Comment by pentschev** [comment]
> This means we will require to upgrade min version to `2022.03.1` as opposed to `2022.02.1`?

Sorry I missed responding here. Yes, we'll need https://github.com/dask/distributed/pull/5904 merged first, but likely this will be in `2022.03.0`.

**Comment by jakirkham** [comment]
PR ( https://github.com/dask/distributed/pull/5904 ) just got merged, which I think fixes the issue here. The plan is to release Dask today. So hopefully that should unblock us here

File: conda/recipes/dask-cuda/meta.yaml [hunk 1]
```diff
@@ -27,8 +27,8 @@ requirements:
     - setuptools
   run:
     - python
-    - dask>=2021.11.1
-    - distributed>=2021.11.1
+    - dask>=2022.02.1
+    - distributed>=2022.02.1
```
Comment (galipremsagar) [inline conda/recipes/dask-cuda/meta.yaml:31]:
```suggestion
    - dask>=2022.03.0
    - distributed>=2022.03.0
```

File: requirements.txt [hunk 2]
```diff
@@ -1,5 +1,5 @@
-dask>=2021.11.1
-distributed>=2021.11.1
+dask>=2022.02.1
+distributed>=2022.02.1
```
Comment (galipremsagar) [inline requirements.txt:2]:
```suggestion
dask>=2022.03.0
distributed>=2022.03.0
```

**Comment by pentschev** [comment]
rerun tests

**Comment by pentschev** [comment]
rerun tests

**Review by pentschev — APPROVED** [review]
Everything's green again, thanks @galipremsagar !

**Pull request merged**